src/LinuxTracingIntegrationTests/CMakeLists.txt

```diff
@@ -3,32 +3,47 @@
 # found in the LICENSE file.
 
 cmake_minimum_required(VERSION 3.15)
-
 project(LinuxTracingIntegrationTests)
 
-add_library(LinuxTracingIntegrationTestPuppetSharedObject SHARED)
 
-target_sources(LinuxTracingIntegrationTestPuppetSharedObject PRIVATE
-        LinuxTracingIntegrationTestPuppetSharedObject.cpp)
+add_library(IntegrationTestPuppetSharedObject SHARED)
+
+target_sources(IntegrationTestPuppetSharedObject PRIVATE
+        IntegrationTestPuppetSharedObject.cpp)
 
-target_link_libraries(LinuxTracingIntegrationTestPuppetSharedObject PUBLIC
+target_link_libraries(IntegrationTestPuppetSharedObject PUBLIC
         dl
         OrbitBase)
 
-strip_symbols(LinuxTracingIntegrationTestPuppetSharedObject)
+strip_symbols(IntegrationTestPuppetSharedObject)
+
+
+add_library(IntegrationTestCommons STATIC)
```

Comment:
Do we want to split the different modules into different directories?

Reply:
I would like to avoid that. I need multiple targets because I want/need `LinuxTracingIntegrationTests` and the upcoming `OrbitServiceIntegrationTests` to be different binaries. So that brought me to define a third common target. But some code in this directory works closely together (e.g., the puppet actions together with the test code that verifies those actions) so I would like not to scatter it across multiple directories, I think it will hurt readability.

Reply:
Okay, fine for me :)